### ISSUE TITLE
Post Chrome 55.0.2883.95 update works in Chrome inspect but not device inspector

### ISSUE THREAD

**Issue by bmwertman** [mauricecruz/chrome-devtools-zerodarkmatrix-theme#177]
Following an update to Chrome 55.0.2883.95 today No themes are applied to the device inspector window. 

I have tried uninstalling and reinstalling the extension and going through the setup procedure and relaunching Chrome but the issue persists. I am on the latest build of the extension (2.2.4).

**Comment by mauricecruz** [comment]
Hmm I have it loading on the same version. In my experience, I've notice that when I update chrome sometimes, it turns off some extensions. You may have tried that so other things to check is if UI Themes experiments is checked off.

**Comment by mauricecruz** [comment]
Is this still an issue?

**Comment by bmwertman** [comment]
Yes, sorry for the slow response. I've just been using it with the default theme. I checked and "Allow custom UI themes" is checked in the experiments. The extension is also enabled. I'm not sure what else to try.

**Comment by bmwertman** [comment]
Hey not sure what changed but my dark theme started working again. One build inspection it didn't work and less than 5 minutes later it did. No idea what changed. But I'll go ahead and close this.

**Comment by mauricecruz** [comment]
@bmwertman thanks for the followup. If you ever figure it out - I'd be interested to know the root cause.

**Comment by HugoMatilla** [comment]
Hi, it is happening to me too.

Chrome version:  56.0.2924.87 (64-bit)
Zero Dark Matrix version: 2.2.7

It used to work until I updated to the chrome version that has a built in dark theme.
I forgot about it, but recently I am trying to make it work again.
I changed the dev tools to default theme, and set up  Zero Dark Matrix. 
In chrome inspect works but not in the device inspector.
Is there any extra step that we should do to make it work?

Thanks.
